hack/multikueue-e2e-test.sh

```diff
@@ -115,7 +75,11 @@ function kueue_deploy {
 
 trap cleanup EXIT
 startup
-kind_load
+prepare_docker_images
+kind_load "$MANAGER_KIND_CLUSTER_NAME" &
+kind_load "$WORKER1_KIND_CLUSTER_NAME" &
+kind_load "$WORKER2_KIND_CLUSTER_NAME" &
```

Comment:
I don't think this will work correctly since we're changing the context every time. I'm wondering if we can avoid switching clusters for each installation – maybe we could specify the cluster directly using `kubectl --context=<context-name> apply -f <file.yaml>` instead?